### PR TITLE
[Flutter Parent][MBL-13654] Reply to a conversation

### PR DESCRIPTION
**Pull request by JordanMarshall**
Also moves the message item UI to its own MessageWidget

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018624&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/screens/inbox/reply/conversation_reply_interactor.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019 - present Instructure, Inc.
```
Comment (Blackchai) [inline apps/flutter_parent/lib/screens/inbox/reply/conversation_reply_interactor.dart]: Obviously not a big deal, but we should probs update all of these newly licensed files to 2020.

File: apps/flutter_parent/lib/screens/inbox/reply/conversation_reply_screen.dart [hunk 2]
```diff
@@ -0,0 +1,248 @@
+// Copyright (C) 2019 - present Instructure, Inc.
```
Comment (Blackchai) [inline apps/flutter_parent/lib/screens/inbox/reply/conversation_reply_screen.dart]: 2020

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018770&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**